sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/codegen/GenerateOrdering.scala

```diff
@@ -118,7 +118,42 @@ object GenerateOrdering extends CodeGenerator[Seq[SortOrder], Ordering[InternalR
           }
       """
     }.mkString("\n")
-    comparisons
+
+    /*
+     * 40 = 7000 bytes / 170 (around 170 bytes per ordering comparison).
```

Comment:
Ah sorry this is vague.

Actually I ran all the 36 test cases in `OrderingSuite` and logged each of the generated `comparisonXXX` method's size. Each method:
- ＜ 4.9 KB when `numberOfComparisonsThreshold = 40`
- ＜ 6.4 KB when `numberOfComparisonsThreshold = 50`
- ＜ 7.8 KB when `numberOfComparisonsThreshold = 60`

I thought it'd be safer if we pick 40 (taking minor future changes into account). Thus 170 should be considered as some kind of a safe assumption (or not?).

Would you share your thoughts on this?